### PR TITLE
update dependencies, use absolute file url in hero images

### PR DESCRIPTION
**Pull request by wibjorn**
Preview: preview-473

Uses readme steps to update deps. Also runs audit fix.

Next leap will be to 16.17, but not quite ready yet.

## Testing

1. Builds should pass.
2. Review site should be available.

### CODE REVIEW

File: site/src/patterns/hero.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ Hero with a theme-aware companion image.
 <section
     class="hero hero-image background-color-body-accent gradient-border-right gradient-border-body-accent"
     style="
-        --hero-background-image-light: url('~/src/scaffold/media/hero-400-light.jpg');
-        --hero-background-image-dark: url('~/src/scaffold/media/hero-400-dark.jpg');
+        --hero-background-image-light: url('https://learn.microsoft.com/en-us/media/home-and-directory/home-hero_light.png?branch=main');
+        --hero-background-image-dark: url('https://learn.microsoft.com/en-us/media/home-and-directory/home-hero_dark.png');
```
Comment (ollips) [inline site/src/patterns/hero.md:40]: `?branch=main` is missed

File: site/src/patterns/hero.md [hunk 2]
```diff
@@ -36,8 +36,8 @@ Hero with a theme-aware companion image.
 <section
     class="hero hero-image background-color-body-accent gradient-border-right gradient-border-body-accent"
     style="
-        --hero-background-image-light: url('~/src/scaffold/media/hero-400-light.jpg');
-        --hero-background-image-dark: url('~/src/scaffold/media/hero-400-dark.jpg');
```
Comment (ollips) [inline site/src/patterns/hero.md:40]: Remove the physical files as well 🙏🏻

File: site/src/patterns/hero.md (hunk 1 above)
Comment (ollips) [reply site/src/patterns/hero.md:40]: Actually, seems like it's not needed at all. Remove from the light image path then?

**Comment by homing1** [comment]
I noticed a vulnerability after pulling the branch and `npm i`. Is it something that can be fixed? 
image

File: site/src/patterns/hero.md (hunk 1 above)
Comment (wibjorn) [reply site/src/patterns/hero.md:40]: updated

File: site/src/patterns/hero.md (hunk 2 above)
Comment (wibjorn) [reply site/src/patterns/hero.md:40]: updated

**Review by ollips — APPROVED** [review]

**Review by homing1 — APPROVED** [review]

**Pull request merged**